### PR TITLE
Fix blue Sturdy Pyxis crashing server

### PR DESCRIPTION
**Pull request by Omnione**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by Omnione** [comment]
fix for issue #6

File: scripts/globals/sturdypyxis.lua [hunk 1]
```diff
@@ -2205,7 +2205,7 @@ end
     ----------------------------------------------------------------------
     -- Desc: Despawn a chest and reset its local var's
     ----------------------------------------------------------------------
-function RemoveChest(player, npc, addcruor, delay)
+function RemoveChest(npc, addcruor, delay)
     local zoneId = player:getZoneID()
```
Comment (Lynnea1320) [inline scripts/globals/sturdypyxis.lua:2209]: player is not defined

Comment (Omnione) [reply scripts/globals/sturdypyxis.lua:2209]: ok i will fix that now, my mistake.

Comment (Omnione) [reply scripts/globals/sturdypyxis.lua]: all done

**Pull request merged**